src/firefly/js/visualize/ui/ImageSelectPanel.jsx

```diff
@@ -0,0 +1,533 @@
+            this.setState({currentCatalogIdx: crtCatalogId, fields});
+        }
+    }
+
+
+    render() {
+        return <ImageSelectionView {...this.state}/>;
+
+    }
+}
+
+/**
+ *  presentation container for image select panel
+ *  @param {number} currentCatalogIdx
+ */
+class ImageSelectionView extends Component {
+
+    constructor(props) {
+        super(props);
+        this.state = props;
+    }
+
+    componentWillReceiveProps(props) {
+        this.state = props;
+    }
+
+    changeCatalog(index) {
+        this.setState({currentCatalogIdx: index});
+    }
+
+    render() {
+
+        // tabs for each catalog
+        var categoryTabs = panelCatalogs.map((item, index) =>
+             (<Tab key={index} name={item.Title}>
+                  <CatalogTabView catalog={item} />
+             </Tab>)
+        );
+
+        var helpId = 'basics.catalog';
+
+        /*
+         * top: target panel
+         * middle: tab panel for catalogs
+         * bottom: size panel
+         *
+         * Load button and help icon
+         */
+        return (
+            <div >
+            <FieldGroup  groupKey={panelKey} reducerFunc={ImageSelPanelChange} keepState={true}>
+                <table className={'imagepanel'}>
+                    <tbody>
+                        <tr>
+                            <td> <TargetPanelSetView /> </td>
+                        </tr>
+                        <tr>
+                            <td>
+                                <Tabs onTabSelect={this.changeCatalog.bind(this)} defaultSelected={1} >
+                                    {categoryTabs}
+                                </Tabs>
+                            </td>
+                        </tr>
+                        <tr>
+                            <td className={'sizerow'}>
+                                <div className={'sizerow'}>
+                                    <ImageFeaturesView {...this.state} />
+                                </div>
+                            </td>
+                        </tr>
+                        <tr>
+                            <td className={'okrow'}>
+                                <div className={rightpadding}>
+                                    <HelpIcon helpId={helpId}/>
+                                </div>
+                                <div className={rightpadding} >
+                                    <CompleteButton
+                                        groupKey={panelKey}
+                                        onSuccess={resultSuccess}
+                                        text={'Load'}
+                                        dialogId={popupId}
+                                    />
+                                </div>
+                             </td>
+                        </tr>
+                    </tbody>
+                </table>
+            </FieldGroup>
+            </div>
+        );
+     }
+}
```

Comment:
We are trying to get away from tables for Layout.  Using table to layout a table is fine but we are trying to use divs for other layout.  Since I am out today, Loi could probably give you an overview of the best techniques we have been using. 

- display:'inline-block'
- display: 'flex'

It will be a little more work at first but cleaner in the end.